pom.xml

```diff
@@ -275,6 +275,11 @@
 				<artifactId>lucene-queries</artifactId>
 				<version>4.10.4</version>
 			</dependency>
+			<dependency>
+				<groupId>org.apache.lucene</groupId>
+				<artifactId>lucene-analyzers-phonetic</artifactId>
+				<version>4.10.4</version>
+			</dependency>
```

Comment:
Are we just adding these here so they are available and users that want to use phonetic mappings don't have to add them to their own modules?  I think we could just add this in PIH Core?

Reply:
That's what I thought, and then I removed it, and the build broke. I don't know exactly what it's providing.

Reply:
That's bit of a red flag... :)

Reply:
What's the build failure message if you do not include that? Is it failing to compile or some tests failing? Could you please paste the full output?

Reply:
Whoops, I looked at the output and realized it's that I had conflicting POMs. I've removed the dependency.